Core/Src/datalogging/odometer_calc.c

```diff
@@ -49,8 +50,11 @@ void odometer_calc() {
         if ((speed_var.wheels[FRONT_RIGHT] && speed_var.wheels[FRONT_LEFT]) != 0) {
```

Comment:
aqui é pra fazer a verificação explícita
```suggestion
        if (speed_var.wheels[FRONT_RIGHT] !=  0 && speed_var.wheels[FRONT_LEFT] != 0) {
```

Reply:
o jeito que tava funciona, mas ainda não é 100% direto na intenção do código

Reply:
Beleza